### PR TITLE
Minor text mistakes

### PR DESCRIPTION
**Pull request by YuriySamorodov**

### CODE REVIEW

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

**Comment by scanum** [comment]
[@JohanFreelancer9](https://github.com/JohanFreelancer9) Copy editing is needed for this PR. Thanks.

File: Exchange/ExchangeOnline/mailbox-migration/perform-gspace-migration-powershell.md [hunk 1]
```diff
@@ -72,7 +72,5 @@ ms.custom: seo-marvel-jun2021
 
 To learn more about:
 
-- Completion of migration batch, see [Completion of migration batch in PowerShell](completion-gspae-migration-batch-powershell.md).
-
-- How the migration happens in backend, see [Overview of the process](how-it-all-works-in-the-backend.md).
-- 
\ No newline at end of file
+- Completion of migration batch, see [Completion of migration batch in PowerShell](completion-gspae-migration-batch-powershell.md)
+- How the migration happens in backend, see [Overview of the process](how-it-all-works-in-the-backend.md)
```
Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-gspace-migration-powershell.md:76]:
```suggestion
- Completion of migration batch, see [Completion of migration batch in PowerShell](completion-gspae-migration-batch-powershell.md)
- How the migration happens in the backend, see [Overview of the process](how-it-all-works-in-the-backend.md)
```

Comment (JohanFreelancer9) [inline Exchange/ExchangeOnline/mailbox-migration/perform-gspace-migration-powershell.md:76]:
```suggestion
- How the migration happens in the backend, see [Overview of the process](how-it-all-works-in-the-backend.md)
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @YuriySamorodov, kindly commit the minor additional suggested change. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by JoanneHendrickson** [comment]
@YuriySamorodov   Can you please select the checkbox on your branch "Allow edits from maintainers" otherwise I cannot merge this PR.

**Comment by YuriySamorodov** [comment]
> @YuriySamorodov Can you please select the checkbox on your branch "Allow edits from maintainers" otherwise edit or merge this PR. thanks

Sorry about that. This is completed

**Comment by scanum** [comment]
@tiburd PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**